src/components/organisms/LoginForm.tsx

```diff
@@ -0,0 +1,82 @@
+import { AuthService } from '@src/services';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Space } from '../atoms';
+export default function LoginForm() {
+  const [loginFormState, setLoginFormState] = useState({
+    email: '',
+    password: '',
+  });
+
+  const handleLoginFormChange = (e) => {
+    if (/[ㄱ-ㅎ|ㅏ-ㅣ|가-힣]/.test(e.target.value)) {
+      alert('한글 입력은 지원하지 않습니다. 영문으로 전환해주세요.');
+      focus;
+      return;
+    }
+    setLoginFormState({ ...loginFormState, [e.target.name]: e.target.value });
```

Comment:
```suggestion
  const handleLoginFormChange = (e) => {
    const {name, value} = e.target;
  
    if (/[ㄱ-ㅎ|ㅏ-ㅣ|가-힣]/.test(value)) {
      alert('한글 입력은 지원하지 않습니다. 영문으로 전환해주세요.');
      focus;
      return;
    }
    setLoginFormState({ ...loginFormState, [name]: value });
```

destructuring을 적용할 수 있는 곳엔 최대한 적용해주세요!

Reply:
bd7c002